### PR TITLE
Fix cyclomatic complexity

### PR DESCRIPTION
**Pull request by EGiataganas**

### CODE REVIEW

File: spec/controllers/webhooks_controller_spec.rb [hunk 1]
```diff
@@ -122,13 +121,13 @@ def bad_encoding(email)
...
-    before :each do
+    before do
       expect(controller).to receive(:authorized?).and_return(false)
     end
 
-    it "returns a 403 unauthorized status" do
+    specify do
       post :email, authorized(email_from_prison)
-      expect(response.status).to eq(403)
+      expect(response).to have_http_status(:forbidden)
```
Comment (SteveMarshall) [inline spec/controllers/webhooks_controller_spec.rb]: :+1:

File: lib/email_validator.rb [hunk 2]
```diff
@@ -14,6 +14,7 @@ def validate(record)
 
   private
 
+  # rubocop:disable Metrics/CyclomaticComplexity
```
Comment (SteveMarshall) [inline lib/email_validator.rb]: Why're we disabling the check here? Wouldn't it make more sense to follow the pattern you use in `WebhooksController`?

Comment (SteveMarshall) [inline lib/email_validator.rb]: As @dan-palmer and I discussed, I'm happy to leave this for #276.

Comment (EGiataganas) [inline lib/email_validator.rb]: @SteveMarshall :+1:

**Pull request merged**